[v3] -tags server build fails (BrowserWindow missing Window.SetScreen)

Building any Wails v3 application with `-tags server` fails to compile in v3.0.0-alpha.78. The `BrowserWindow` type (defined in `v3/pkg/application/browser_window.go` under `//go:build server`) does not implement the `Window` interface — it's missing `SetScreen(screen *Screen) Window`, which was added to the `Window` interface in #5067.

The bug is also present on the current tip of `v3-alpha`.

### To Reproduce

Any v3 example, e.g. from a fresh checkout:

```bash
cd v3/examples/dock
go build -tags server .

Errors (truncated)
v3/pkg/application/browser_window.go:65:80: cannot use b (variable of type *BrowserWindow) as Window value in return statement: *BrowserWindow does not implement Window (missing method SetScreen)
v3/pkg/application/browser_window.go:74:80: cannot use b (variable of type *BrowserWindow) as Window value in return statement: ...
```

### Expected behavior

`go build -tags server .` succeeds, producing a server-mode binary as documented in `pkg/application/application_server.go.`

### Suggested fix

Add a no-op SetScreen to BrowserWindow that matches the existing no-op pattern in the file (browser windows have no notion of physical screens):

```
func (b *BrowserWindow) SetScreen(screen *Screen) Window { return b }
```

### Why this matters / use case

Server mode is the documented path for running the same Wails app under HTTP for headless test harnesses (Playwright/etc.) and for future hosted deployment. With it broken, projects pinned to alpha.78 cannot use the published task build:server / task run:server workflow without a local fork.

### Environment

Wails CLI Version: v3.0.0-alpha.78
Platform: macOS (Darwin 24.6.0); also reproduces on Linux/Windows because the offending file is platform-agnostic
Go: any recent 1.22+

--- a/v3/pkg/application/browser_window.go
+++ b/v3/pkg/application/browser_window.go
@@ -65,6 +65,7 @@ func (b *BrowserWindow) ForceReload()                                 {}
 func (b *BrowserWindow) Fullscreen() Window                           { return b }
 func (b *BrowserWindow) GetBorderSizes() *LRTB                        { return nil }
 func (b *BrowserWindow) GetScreen() (*Screen, error)                  { return nil, nil }
+func (b *BrowserWindow) SetScreen(screen *Screen) Window              { return b }
 func (b *BrowserWindow) GetZoom() float64                             { return 1.0 }
 func (b *BrowserWindow) handleDragAndDropMessage(filenames []string, dropTarget *DropTargetDetails) {}
 func (b *BrowserWindow) InitiateFrontendDropProcessing(filenames []string, x int, y int) {}

